src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesObsolete.TableExt.al

```diff
@@ -85,18 +85,20 @@ tableextension 308 NoSeriesObsolete extends "No. Series"
     [Obsolete('The function has been moved to codeunit NoSeriesManagement', '24.0')]
     procedure FindNoSeriesLineToShow(var NoSeriesLine: Record "No. Series Line")
     var
-        NoSeriesMgt: Codeunit NoSeriesMgt;
+        NoSeriesManagement: Codeunit NoSeriesManagement;
     begin
-        NoSeriesMgt.FindNoSeriesLineToShow(Rec, NoSeriesLine)
+        NoSeriesManagement.FindNoSeriesLineToShow(Rec, NoSeriesLine)
     end;
 
-    [Obsolete('The event has been moved to codeunit NoSeriesManagement', '24.0')]
+    [Obsolete('The event has been removed.', '24.0')]
+    // Symbol usage indicates no subscribers.
     [IntegrationEvent(false, false)]
     internal procedure OnBeforeValidateDefaultNos(var NoSeries: Record "No. Series"; var IsHandled: Boolean)
     begin
     end;
 
-    [Obsolete('The event has been moved to codeunit NoSeriesManagement', '24.0')]
+    [Obsolete('The event has been removed.', '24.0')]
+    // Symbol usage indicates no subscribers.
```

Comment:
Please state whether there is a replacement. If there is, please name it.

Reply:
there is no replacement. Our data shows there is no usage of these events. I asked the requestor of the events and they use it on-prem only. We are providing them with an alternative way of dealing with their requirement.